src/index.ts

```diff
@@ -56,28 +62,40 @@ export interface GetCallOptions extends CallOptions {
   autoCreate?: boolean;
 }
 
-export interface PushConfig {
-  pushEndpoint: string;
-  attibutes?: Map<string, string>;
+export interface Inventory {
+  callbacks?: Array<RequestCallback<string>>;
+  queued?: Array<{}>;
```

Comment:
`callbacks` and `queued` should not be optional, they will always be set.

Reply:
done